### DISCUSSION TITLE
No `_content.scss` file for blue theme

### DISCUSSION THREAD

**Discussion by JeremiasCortes** [twbs/bootstrap#39931] [Q&A] [answered]
![Anotación 2024-04-29 140243](https://github.com/twbs/bootstrap/assets/127408687/349ea96f-92dd-468c-a305-7fe8bf22ab27)

Para incluir nuevos modos de colores indica en la documentación que se hace en el archivo de _content.scss

Sin embargo ese archivo cuando te has descargado BootStrap 5.3 en mi caso (con npm) no se encuentra ese archvivo
![Anotación 2024-04-29 140837](https://github.com/twbs/bootstrap/assets/127408687/23e5a778-557a-4dc2-98e1-ce68b3a0feb7)

He encontrado que se tiene que hacer en el archivo _navbar.scss

Si me equivoco que alguien me lo explique estaré encantado de recibir orientación sobre esto.

**Comment by julien-deramond** [comment, marked as answer]
Hello @JeremiasCortes 

`site/assets/scss/_content.scss` is just a reference used in our documentation, it's not in the Bootstrap Sass files, not in the package you download. This is rather an example of something that you can write yourself if you need to create a “blue theme”; which is done with the selector `data-bs-theme="blue"` in your custom Sass or CSS file.

**Reply by JeremiasCortes** [reply]
Comprendo, muchas gracias. No sabía con exactitud sobre ello. Que tenga usted un buen día.
